qpdflib/pdfview/locale/ko/viewer.properties

```diff
@@ -65,6 +65,20 @@ cursor_text_select_tool_label=텍스트 선택 도구
 cursor_hand_tool.title=손 도구 활성화
 cursor_hand_tool_label=손 도구
 
+scroll_vertical.title=세로 스크롤 사용
+scroll_vertical_label=세로 스크롤
+scroll_horizontal.title=가로 스크롤 사용
+scroll_horizontal_label=가로 스크롤
+scroll_wrapped.title=포장된 스크롤 사용
+scroll_wrapped_label=포장된 스크롤
+
+spread_none.title=한 페이지로 보기
+spread_none_label=한 페이지
+spread_odd.title=홀수 페이지를 기준으로 두 페이지씩 보기
+spread_odd_label=홀수 두 페이지
+spread_even.title=짝수 페이지를 기준으로 두 페이지씩 보기
+spread_even_label=짝수 두 페이지
```

Comment:
I see these properties are only defined for en-US locale. Does this mean that other languages are also affected?

Reply:
Is this the same issue as #33 ?

Reply:
> I see these properties are only defined for en-US locale. Does this mean that other languages are also affected?

Yes, that's right. Experiments in other languages show the same symptoms.

Reply:
> Is this the same issue as #33 ?

It has been analyzed that the log message of the issue is output in relation to the l10n JavaScript library in QtWebEngineView, but it is not the same error as my pull request, and it seems that it was probably caused by the build process excluding the locale folder.

Reply:
> > I see these properties are only defined for en-US locale. Does this mean that other languages are also affected?
> 
> Yes, that's right. Test in other local show the same symptoms.

I modified that part in my [repository](https://github.com/jjsh0503/qpdf) where I forked your [qpdf](https://github.com/Archie3d/qpdf) and built it and tested it in the same environment, and I didn't get that error message.

Reply:
> > Is this the same issue as #33 ?
> 
> It has been analyzed that the log message of the issue is output in relation to the l10n JavaScript library in QtWebEngineView, but it is not the same error as my pull request, and it seems that it was probably caused by the build process excluding the locale folder.

Or it may be another locale that is not included in the [file](https://github.com/Archie3d/qpdf/tree/master/qpdflib/pdfview/locale).

Reply:
I see, thanks!